tests/integration/acp/relationship/doc_actor/add/with_only_delete_test.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package test_acp_relationship_doc_actor_add
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestACP_OwnerGivesDeleteAccessToAnotherActorWithoutExplicitReadPerm_OtherActorCanDelete(t *testing.T) {
```

Comment:
thought: Lets please try to avoid adding unnecessary empty lines. This makes reading the test very painful.

Reply:
What extra empty lines are you talking about?

Reply:
You add them every other line in the test declaration, it really hinders readability in my opinion but I've avoided commenting about it as it is a personal preference thing and you seem to like it...  If everybody else dislikes it I'd be in favour of banning this though.

Reply:
> you seem to like it

It's a habit more than liking it. As you mentioned it's a personal preference, if there is consensus we can find a tool or linter that does the additional formatting on top of gofmt.

However I am curious though what is the ideal preference people have?
- No space at all between elements in the `[]any{}` and no space in the args of the action structs?
- No space at all between elements in the `[]any{}` but space in the args of the action structs?
- Space between elements in the `[]any{}` but no space in the args of the action structs?

Reply:
> No space at all between elements in the []any{} and no space in the args of the action structs?

This one for me, and I think the majority of our tests are written like this.

Reply:
Same as Andy for me.